### DISCUSSION TITLE
Alter blockquote styling

### DISCUSSION THREAD

**Discussion by stockphrase** [jpanther/congo#482] [Q&A]
Can someone point me to the place where I can turn off the quotation marks that are automatically placed on block quotes?

**Comment by jpanther** [comment]
Blockquote styling comes from the [Tailwind Typography](https://tailwindcss.com/docs/typography-plugin) plugin. You can either use custom CSS, the typography modifier classes in your templates, or edit the Tailwind configuration to adjust the behaviour. The latter 2 options will require you to rebuild the theme CSS though, as they will adjust the base Tailwind classes that ship with the theme.
